src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css

```diff
@@ -577,8 +577,7 @@ h1.hidden {
     margin-right:10px;
     display:inline-block;
 }
-div.block div.deprecation-comment, div.block div.block span.emphasized-phrase,
-div.block div.block span.interface-name {
+div.block div.deprecation-comment, div.block div.block span.emphasized-phrase {
```

Comment:
A quick search through the code and generated docs suggests that `emphasized-phrase` is unused as well. Should this also be removed?

Reply:
Let me investigate a test program  to compare the class names in the style sheet and the members of HtmlStyle, and then decide whether to merge that here or do a separate commit/PR.

Reply:
Test program reveals differences that need to be investigated. 
See https://bugs.openjdk.java.net/browse/JDK-8267709

For now, I've just removed `emphasized-phrase`